beets/autotag/__init__.py

```diff
@@ -45,13 +45,13 @@ def apply_item_metadata(item, track_info):
     if track_info.data_source:
         item.data_source = track_info.data_source
 
-    if track_info.lyricist is not None:
+    if track_info.get('lyricist'):
```

Comment:
To minimize changes like this, we could have `__getattr__` return `None` by default for missing fields. Not very Pythonic, but could be fine for these classes…